test/controllers/trust.historical.beneficial.owner.controller.spec.ts

```diff
@@ -276,5 +280,69 @@ describe('Trust Historical Beneficial Owner Controller', () => {
       expect(authentication).toBeCalledTimes(1);
       expect(hasTrustWithIdRegister).toBeCalledTimes(1);
     });
+
+    test('renders the current page with end date error message', async () => {
+      // Given
+      mockHistBORequest.endDateDay = "";
```

Comment:
if you modify mockHistBORequest here it will affect the other tests that use it, so you should clone the object for your test first and then modify it.
const copy = { ...mockHistBORequest }
copy.endDateDay =

Reply:
unless you reset the mockHistBORequest in a beforeEach() , that's another option

Reply:
Cloned.